libcxx/include/__algorithm/stable_sort.h

```diff
@@ -106,7 +106,7 @@ _LIBCPP_HIDE_FROM_ABI void __merge_move_construct(
 }
 
 template <class _AlgPolicy, class _Compare, class _InputIterator1, class _InputIterator2, class _OutputIterator>
-_LIBCPP_HIDE_FROM_ABI void __merge_move_assign(
+_LIBCPP_HIDE_FROM_ABI _LIBCPP_CONSTEXPR_SINCE_CXX26 void __merge_move_assign(
```

Comment:
Perhaps we should locate the bug causing CI failure and add some workaround for GCC 14, although I have no reduced example yet.

Reply:
For record: the bug is likely to be https://gcc.gnu.org/bugzilla/show_bug.cgi?id=115754 which is resolved in GCC 14.2.

Reply:
Thank you for the investigation! I was wondering already why I could not reproduce it with my local gcc14.2. Will try out your suggestion for the workaround.

Reply:
Alternative: Wait until ci updates to gcc14.2. This is the current latest stable release and as of libcxx.llvm.org that is the only one we are required to support.